subprojects/internal-integ-testing/internal-integ-testing.gradle.kts

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile(project(":cli"))
     compile(project(":launcher"))
     compile(project(":native"))
+    compile(project(":resourcesHttp"))
```

Comment:
Can we offer the test fixture in `resourcesHttp`? So we don't have to add the dependency in the global test fixtures module?

Reply:
If I understand you correctly, you're suggesting moving `AbstractHttpDependencyResolutionTest` to `resourcesHttp`. This is a good idea but I'm a bit reluctant to do because that means we have to move all dependencies (the whole https://github.com/gradle/gradle/tree/42247525e4851cd9346a9ff535b2e8ded7b5f6a2/subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http package to `resourcesHttp`).

Reply:
Maybe I misunderstood you and you had better ideas.

Reply:
@blindpirate Why do you need to move all the dependencies of `AbstractHttpDependencyResolutionTest` to `resourcesHttp`? `resourcesHttp` already depends on `internal-integ-testing`, right?

The only thing you would need to do is to create a `testFixtures` source set in `resourcesHttp` and move the `AbstractDependencyResolutionTest` there. Then everybody using this class would need to depend on the test fixtures of `resourcesHttp`, too.

This doesn't have to happen in this PR if it gets too involved, but I still think it would be the right direction.

Reply:
Thank you, I think I've made the change you're suggesting, would you like to take another look?